Uchu.World/Objects/Zone.cs

```diff
@@ -130,7 +130,7 @@ public async Task InitializeAsync()
             Checksum = ZoneInfo.LuzFile.GenerateChecksum(ZoneInfo.LvlFiles);
             ZoneId = (ZoneId) ZoneInfo.LuzFile.WorldId;
             SpawnPosition = ZoneInfo.LuzFile.SpawnPoint;
-            SpawnRotation = ZoneInfo.LuzFile.SpawnRotation;
+            SpawnRotation = new Quaternion(ZoneInfo.LuzFile.SpawnRotation.W, ZoneInfo.LuzFile.SpawnRotation.X, ZoneInfo.LuzFile.SpawnRotation.Y, ZoneInfo.LuzFile.SpawnRotation.Z);
```

Comment:
As the quaternion in LuzFile.SpawnRotation is wrong, this is something that should be fixed in InfectedRose instead of here. Additionally, the correct way to interpret it is actually `Quaternion(Y, Z, W, X)` instead of `Quaternion(W, X, Y, Z)`; it's written in the file as W, X, Y, Z, so for X the second value should be used, for Y the third value, for Z the fourth value and for W the first value.
Supported by some screenshots:
Crux Prime with Quaternion(W, X, Y, Z)
![Crux Prime with Quaternion(W, X, Y, Z)](https://user-images.githubusercontent.com/11255568/126796108-2c45da3b-3c9d-46c8-b8e0-5af0ef0d6d5b.png)
Crux Prime with Quaternion(Y, W, Z, X)
![Crux Prime with Quaternion(Y, W, Z, X)](https://user-images.githubusercontent.com/11255568/126796154-213c9eeb-1520-4768-860f-b1cd70d9fb32.png)

Reply:
Makes sense and thanks for correcting out how the rotation should be corrected.